app/src/main/res/values/strings.xml

```diff
@@ -265,10 +269,10 @@
     <!-- Feature discovery -->
     <string name="app_info_title">How The Algorithm Works</string>
     <string name="app_info_body">
-        1. Choose between two users repeatedly.\n
-        2. The app creates a stack ranking.\n
+        1. You choose between two users repeatedly.\n
+        2. The app then creates a stack ranking.\n
         3. A daily algorithm matches for mutual attraction.\n
-        4. Everyone is limited to the top 10 matches.
+        4. Enjoy your top 10 matches!
```

Comment:
It's great that it's short, but it also seems to lose meaning. The new version leaves open how many total matches the user gets. It could be that the user gets 100 matches and the text suggests to enjoy the first 10. The message of limiting is edited out.

E.g. "Here is your new car for your 18th birthday. Enjoy the first 1,000 miles. They'll make a great memory." vs "Your warranty runs out after 1,000 miles.

Reply:
Tried to limit character count here and reword #4.